thorlcr/activities/thdiskbaseslave.cpp

```diff
@@ -556,6 +556,8 @@ void CDiskWriteSlaveActivityBase::gatherActiveStats(CRuntimeStatisticCollection
 {
     PARENT::gatherActiveStats(activeStats);
     mergeStats(activeStats, outputIO, diskWriteRemoteStatistics);
+    if (tmpUsage && outputIO) // Update tmpUsage file size (Needed to calc inter-graph spill stats)
+        tmpUsage->setSize(outputIO->getStatistic(StSizeDiskWrite));
```

Comment:
I didn't imagine it :) - there is still a tmpUsage->setSize call left in CDiskWriteSlaveActivityBase::processDone, that should be removed afaics.
It shouldn't be necessary in processDone, because this (gatherActiveStats) will already be called at end of subgraph.

Reply:
sorry, I missed that.  Removed now.